src/sagemaker_huggingface_inference_toolkit/transformers_utils.py

```diff
@@ -231,15 +231,18 @@ def infer_task_from_model_architecture(model_config_path: str, architecture_inde
         architecture = config.get("architectures", [None])[architecture_index]
 
     task = None
+    if "HF_TASK" in os.environ:
+        return os.environ["HF_TASK"]
+
```

Comment:
this might cause the tests to fail

Reply:
Those tests don't seem to be exercise this path or this method...

Reply:
Also this fix is similar to what is done in handler_service.py

```
        # gets pipeline from task tag
        if "HF_TASK" in os.environ:
            hf_pipeline = get_pipeline(task=os.environ["HF_TASK"], model_dir=model_dir, device=self.device)
```